### DISCUSSION TITLE
IS appwrite lite still available?

### DISCUSSION THREAD

**Discussion by donix18** [appwrite/appwrite#2743] [Q&A]
Hi all,
I'm trying to install appwrite lite but it seems the image isn't available anymore. Does it means I have to bring to appwrite one and remove other services? IN this case what env variable and services are mandatory?
Many thanks

**Comment by gewenyu99** [comment]
Hi @donix18. Appwrite Lite was originally created to be used on Heroku. There were some roadblocks we hit, such as removing services like InfluxDB from the stack, which we never fully resolved. At the moment, our resources are focused on other parts of Appwrite, so Appwrite Lite isn't getting the love and attention it deserves. 

For now, you can still disable unwanted services in the Appwrite console, but we are aware it's not as lightweight.

**Comment by gaby** [comment]
I would love to use appwrite-lite. The regular version has way too many containers. For my use case I only need the users/functions/database. 

Maybe the community can help bring appwrite-lite back to life?
